ImmunityReborn/Assets/_TeamAmnesia/Scripts/MemoryAttack.cs

```diff
@@ -59,8 +57,10 @@ private void MyInput()
 
         if ( readyToShoot && shooting && StateMachine.MemoryGauge.CurrentMemoryMeterValue >= 50 )
         {
-            //LaunchMemoryAttack();
-            StateMachine.Animator.Play( animationName ); // LaunchMemoryAttack() called via animation event
+            if (StateMachine.CurrentState is not PlayerAttackingState)
```

Comment:
I believe this If statement can be moved above and joined with a &&. 
Not too important we can get to it after beta.